web-wallet/src/style/main.css

```diff
@@ -121,14 +121,9 @@ svg {
     .dusk-textbox:read-write:enabled,
     *[role="menuitem"]
   ):focus {
-  border-color: color-mix(
-    in srgb,
-    var(--secondary-color-variant-dark) 80%,
-    transparent
-  );
-  box-shadow: 0 0 0.5em
-    color-mix(in srgb, var(--secondary-color-variant-dark) 60%, transparent);
-  outline: 0 none;
+  border-color: var(--secondary-color-variant-dark);
+  outline: var(--control-border-size) solid var(--secondary-color-variant-dark);
+  outline-offset: -0.125rem;
```

Comment:
There is no reason to lose more time on this right now, but I'd open an issue to investigate the possibility of removing the use of `rem` here.
Probably a specific rule for small variants if the issue is caused by them using a different font-size? Or maybe anyway?

Reply:
The use of `rem` here was to fix the issues that appeared on Firefox with the border as `em` could not be used. The other alternative would have been the unit to be in `px`.

Reply:
https://github.com/dusk-network/rusk/issues/1538